app/controllers/posts_controller.rb

```diff
@@ -25,11 +26,16 @@ def show
 
 	def edit
 		@post = Post.find(params[:id])
+
+		if is_current_user_owner(@post)
+			render 'edit'
+		end
 	end
 
 	def update
 		@post = Post.find(params[:id])
 
+		authorize @post
```

Comment:
I'm getting this error here and need to investigate why:
undefined method `authorize' for #<PostsController:0x007ffd26cef510

Reply:
Resolved this issue, needed to "include Pundit" in PostsController. That step wasn't detailed in the pundit gem README, but realized after thanks to stackoverflow :)